gamification-evm-services/src/main/java/io/meeds/evm/gamification/service/EvmTriggerService.java

```diff
@@ -117,7 +117,7 @@ private void processEvent(EvmTrigger evmTrigger) {
                               + Utils.MIN_AMOUNT + ": " + evmTrigger.getAmount() + "}";
         broadcastEvmEvent(evmTrigger.getTrigger(),
                           receiverId,
-                          evmTrigger.getTransactionHash(),
+                          evmTrigger.getNetworkId() + evmTrigger.getTransactionHash(),
```

Comment:
Please add a separator like `@` or `#` to be able to read the value easily

Reply:
done!